src/components/Config/Config.tsx

```diff
@@ -18,7 +18,10 @@ function Config() {
   const [confirmationTime, setConfirmationTime] = useState(ConfirmationTime.oneDay);
   const [submitted, setSubmitted] = useState(false);
   const [completed, setCompleted] = useState(false);
+  const [addedConnetionModalDisplay, setAddedConnetionModalDisplay] = useState(false);
   const [network, setNetwork] = useState(Network.none);
+  const localStorage = window.localStorage;
```

Comment:
localStorage is a static property so it doesn't make sense to access it on every render. if you want localStorage then just get it at the top of the file, not inside the component.
Anyway, since this redirect is only a temporary thing (i.e. the information should only be stored while the signing up and then it's not needed anymore), using localStorage is counterintuitive and misleading. localStorage is intended for more permanent things. For temporary stuff, sessionStorage is more appropriate